posts/views.py

```diff
@@ -76,34 +76,33 @@ def dislike(request, id):
         reaction = Reaction()
         reaction.post = post
         reaction.user = user
-    reaction.vote = -1
+    reaction.vote = False
     reaction.save()
     return Response({}, status=status.HTTP_200_OK)
 
 
-@api_view(['GET'])
+@api_view(['DELETE'])
 @permission_classes([p.IsAuthenticated, ])
 def cancel_reaction(request, id):
     post = get_object_or_404(Post, id=id)
     user = request.user
     reaction = Reaction.objects.filter(post=post, user=user).first()
     if reaction:
-        reaction.vote = 0
-        reaction.save()
+        reaction.delete()
     return Response({}, status=status.HTTP_200_OK)
 
 
 @api_view(['GET'])
 @permission_classes([p.AllowAny, ])
 def count_likes(request, id):
```

Comment:
These endpoints do not make sense, and why make 2 separate endpoints for the count of likes and dislikes, if you can make one that will return both